config/deploy/dev.rb

```diff
@@ -0,0 +1,6 @@
+ENV['APP_ENV']  ||= 'integration'
+require_relative 'server_settings'
+
+server 'sdr-services-dev.stanford.edu', user: 'sdr2service', roles: %w{app}
+
+Capistrano::OneTimeKey.generate_one_time_key!
```

Comment:
@jcoyne - the new pattern without the server_settings.rb stuff looks like this:
```ruby
# config/deploy/dev.rb 
ENV['APP_ENV'] ||= 'integration'
set :default_env, 'APP_ENV' => ENV['APP_ENV']
server 'sul-sdr-services-dev.stanford.edu', user: 'sdr2service', roles: %w{app}
Capistrano::OneTimeKey.generate_one_time_key!
```
But the -stage and -prod need the `bundle_without` stuff noted elsewhere.  Also, we need some new symlinks to the new shared-configs files, when the puppet stuff is merged and deployed.